xlators/cluster/dht/src/dht-common.h

```diff
@@ -106,11 +106,11 @@ struct dht_layout {
 typedef struct dht_layout dht_layout_t;
 
 struct dht_stat_time {
-    uint32_t atime;
+    uint64_t atime;
```

Comment:
nit: you might want to consider (to reduce padding), something like:
```
	uint64_t	atime;
	uint64_t	mtime;
	uint64_t	ctime;
	uint32_t	atime_nsec;
	uint32_t	mtime_nsec;
	uint32_t	ctime_nsec;
```
(taken from struct fuse_attr , but similar idea in struct iatt as well)

(also, unrelated, it's quite annoying how the order of the fields between 'a', 'm' and 'c' is different across our code base)